docs/resources/built-in-decorators.md

```diff
@@ -65,3 +65,32 @@ lint:
       tagNames:
         pet: ./my-custom-description.md
 ```
+
+### hide-internals
+
+Using `x-internal` tag by default to hide nodes:
+```
+  PathItem
+  Operation
+  Schema
+  Response
+  RequestBody
+  Example
+  MediaType
+  Server
+  Callback
+  Parameter
+```
+
+The `tagToHide` option uses to define name of tag.
+
+```yaml
+lint:
+  decorators:
+    hide-internals:
+      tagToHide: 'hideit'
+```
+
+### clear-unused-components
```

Comment:
Is this a decorator or a command line argument or both? Also, I would recommend to rename it.
```suggestion
### remove-unused-components
```

Reply:
@adamaltman done.